style.css

```diff
@@ -175,13 +184,254 @@ body {
   transition: 1s ease-out .2s;
 }
 
-.vectormob {
+.vectormob,
+.second-container__bags-mob {
   display: none;
 }
 
-@media (max-width: 1235px) {
+.second {
+  background-color: #FAE8E4;
```

Comment:
Codacy found an issue: [Expected "#FAE8E4" to be "#fae8e4" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)